apps/pm-user/src/pages/signUp/SignUp.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FormWraper, SignUpForm } from '../../components';
+import styles from './signUp.module.scss';
+
+function SignUp() {
+  return (
+    <FormWraper
+      bottomText="Already have account then!"
+      formTitle="SignUp"
+      formDescription="if you dont have account create here!"
+      pageToRedirect="login"
+      pageToRedirectTitle="LogIn"
+      sideImage="signupPage"
+    >
+      <SignUpForm />
+    </FormWraper>
```

Comment:
We  only have one place for using SignUp component.
Hence, bottomText and formDescription props are unnecessary.
Why not just hardcode them?

Reply:
1. using SignUpForm component will keep code clean and understandable.
2. and bottom text and form description are props because they are being used in the loginForm.